pyiode/objects/table.pyx

```diff
@@ -297,10 +304,20 @@ cdef class TableLine:
 
     Attributes
     ----------
+    # GDM>
+    # * I *wonder* if using subclasses per type wouldn't be better (for example if LecCell had additional methods)?
```

Comment:
As I said above, this is a good suggestion but unfortunatly I have not time to do this right now.

Plus, I am trying to stay as close as possible the C and C++ API for the moment. So I will keep the TableLine class as it for the moment.

Reply:
See issue #583